.eslintrc.js

```diff
@@ -51,6 +39,18 @@ module.exports = {
         natural: false,
       },
     ],
+    "@typescript-eslint/ban-types": [
+      "error",
+      {
+        types: {
+          "React.FC": {
```

Comment:
I did `yarn install` and tried to trigger this error by doing:

```tsx
export const Button: React.FC = () => {
  return <button>hi</button>;
};
```

but ESlint didn't error out!

If you've verified that it works, then I think we're good to merge and I'll figure out my IDE later lol